Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "{{project-name}}"
+name = "insert_project_name_here"
```

Comment:
According to https://github.com/confio/cosmwasm-template/issues/7#issuecomment-567897838, this change must be reverted, right?

I did it in order to be able to test my changes.